Foundation/Check/Main.hs

```diff
@@ -67,11 +67,11 @@ filterTestMatching cfg testRoot
     | null (testNameMatch cfg) = Just testRoot
     | otherwise                = testFilter [] testRoot
   where
-    match acc s = or $ fmap (flip isInfixOf currentTestName) $ testNameMatch cfg
+    match acc s = or (flip isInfixOf currentTestName <$> testNameMatch cfg)
       where currentTestName = fqTestName (s:acc)
     or [] = False
     or (x:xs)
-        | x == True = True
+        | x         = True
         | otherwise = or xs
```

Comment:
Isn't this just absolutely standard `or` from the base library? Why replicate it?